nuttx-configs/px4fmu-v4pro/nsh/defconfig

```diff
@@ -627,7 +627,7 @@ CONFIG_BOOT_RUNFROMFLASH=y
 # Boot Memory Configuration
 #
 CONFIG_RAM_START=0x20000000
-CONFIG_RAM_SIZE=327680
```

Comment:
FYI: These are not use in Nuttx on this Soc - I commited the fix upstream in [Nuttx here](https://bitbucket.org/nuttx/nuttx/commits/df9c5b33a01cac9da1786ede3903c575861cc769)